### ISSUE TITLE
Generating the single binary.

### ISSUE THREAD

**Issue by Ward9250** [JuliaComputing/static-julia#33]
In the readme it says that with julia 0.7 a single binary file can be generated. But doing `julia juliac.jl -ve` currently produces a executable and a dylib (I'm on mac). To generate the single binary, do I have to go into juliac.jl and change the mentions of program.c to program2.c, or is more required?

**Comment by ViralBShah** [comment]
Yes - and @lucatrv is going to prepare a branch for that. On the current master this doesn't work yet, and @vtjnash and I will revisit this after 0.7 alpha.

**Comment by lucatrv** [comment]
Sure, I am going to submit a new PR for this, WIP
